src/Validator/HtmlTagValidator.php

```
@@ -0,0 +1,376 @@
+        return "- <fg={$color}>{$level}</> {$prefix}HTML tag inconsistency in translation key `{$key}` - {$inconsistencyText}";
+    }
+
+    public function distributeIssuesForDisplay(FileSet $fileSet): array
+    {
+        $distribution = [];
+
+        foreach ($this->issues as $issue) {
+            $details = $issue->getDetails();
+            $files = $details['files'] ?? [];
+
+            foreach ($files as $fileName => $fileInfo) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$fileInfo'. (Unused Code Rules)

(UnusedLocalVariable)